paymentrails.gemspec

```diff
@@ -1,18 +1,18 @@
-$:.push File.expand_path("../lib", __FILE__)
+$:.push File.expand_path('lib', __dir__)
 
 Gem::Specification.new do |s|
-  s.name = "paymentrails"
```

Comment:
its safe to rename this too since we'll be publishing on the new package: https://rubygems.org/gems/trolley/